scripts/install-systemd-multi-user.sh

```diff
@@ -83,6 +85,13 @@ EOF
 poly_configure_nix_daemon_service() {
     if [ -e /run/systemd/system ]; then
         task "Setting up the nix-daemon systemd service"
+
+        _sudo "to create the nix-daemon tmpfiles config" \
+              cp /nix/var/nix/profiles/default/$TMPFILES_SRC $TMPFILES_DEST
```

Comment:
```suggestion
              ln -s /nix/var/nix/profiles/default/$TMPFILES_SRC $TMPFILES_DEST
```
I think we want that, as IIRC that matches what `systemd link` will do, and also so upgrading Nix upogrades this file too.

Reply:
I picked `ln -sfn`, to make this more idempotent if there's already an existing `$TMPFILES_DEST`. I assume `systemd link` does that too.